py_gtfs_rt_ingestion/tests/test_convert.py

```diff
@@ -374,3 +403,317 @@ def test_bus_trip_updates_file_conversion() -> None:
             assert upper == str(np_df[col].max())
         if lower != "nan":
             assert lower == str(np_df[col].min())
+
+
+def test_schedule_conversion() -> None:
```

Comment:
same test from the old PR.

ensure we can generate parquet tables correctly for schedule gtfs data.